### ISSUE TITLE
OMIM diseases being categorized as genes in SciGraph load

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/dipper#143]
OMIM diseases are being categorized as genes in scigraph through a (hypothesized) relationship with SO:0000704 (gene) or a subclass.

Note this may be a bug in SciGraph or one of the ingested ontologies rather than dipper.  

One example is OMIM:311510: http://geoffrey.crbs.ucsd.edu:9000/scigraph/graph/OMIM:311510

**Comment by kshefchek** [comment]
Looks like NCBI Gene is the culprit, maybe we can simply make all of these dbxrefs?

        <http://www.ncbi.nlm.nih.gov/gene/7489> a owl:Class ;
            rdfs:label "WSN" ;
            OBO:RO_0002525 OBO:CHR_9606chrX ;
            dc:description "Waisman syndrome" ;
            OIO:hasExactSynonym "Waisman syndrome" ;
            OIO:hasRelatedSynonym "BGMR",
           "Waisman syndrome (basal ganglion disorder with mental retardation)" ;
            rdfs:subClassOf OBO:SO_0000704 ;
            owl:equivalentClass OBO:OMIM_311510,
                <http://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id=HGNC:12795> .

**Comment by cmungall** [comment]
ouch, yes, that will be causing it.

**Comment by nlwashington** [comment]
that gene is listed as an "unknown" gene type in NCBI.  i was defaulting these to be of type "gene".  but i can change these to be of type ["sequence feature"](http://sequenceontology.org/browser/current_svn/term/SO:0000110) instead.  does that sound reasonable, @cmungall ?

**Comment by cmungall** [comment]
If sequence_feature, then it will be invisible from places where we expect genes. I would opt for type `gene` as and have other way of indicating its putative status. In fact the fact it is a plain `gene` and not a subtype (coding etc) already indicates the status. But I'm not that familiar with the shape of the data. If there are 10ks of these then it may be odd to cram them into gene as it inflates gene counts etc

**Comment by nlwashington** [comment]
there are ~2600 human "things" in ncbi that are "unknown" type, and another 878 of "other" type.  generally, they are some kind of thing that is associated with a disease or an untyped feature. often is at a known genetic locus (for example 18q21), but the extent of the feature itself isn't known.

HGNC seems to capture many of the ones linked to phenotype as a [heritable phenotypic marker](http://sequenceontology.org/browser/current_svn/term/SO:0001500).   i can certainly add some logic to say that if there is a known cytogenic locus, then type it as a heritable phenotypic marker?

but others aren't genes at all.  for example, the [Epstein Barr virus integration site](http://www.ncbi.nlm.nih.gov/gene/1888) isn't a gene at all.  but it is certainly heritable.

for mouse, "other" can be used to classify deletions (which is odd).  like for [Del(1)6H](http://www.ncbi.nlm.nih.gov/gene/13255) or the same thing at [MGI](http://www.informatics.jax.org/accession/MGI:103584).  and it looks like "unknown" might also be used for phenotype-related loci, like for [anx](http://www.ncbi.nlm.nih.gov/gene/11743)/[MGI](http://www.informatics.jax.org/accession/88029)

human: 2610 uknown; 878 other
zfish:  146 unknown ; 352 other
mouse: 10890 unknown; 2704 other

**Comment by nlwashington** [comment]
so the problem is that if we type things that are otherwise "diseases" as "genes" then we run into the problem as reported by kent above.  how will we otherwise get around the messiness of omim having the same id for a locus and disease?

**Comment by mellybelly** [comment]
geno+SO should address the deletion being a feature that is a variant_of gene, where there may be more than one variant_of to gene triples
but the deletion itself should be typed as a feature or genomic variation, not a gene, of course
We can also curate the omim mess and feed it back to them if that is helpful

**Comment by nlwashington** [comment]
omim, of course, are very aware of the combo gene+disease issue.  i don't see them changing these anytime soon.  so we have to deal with it.  (though i have just emailed Ada with the recommendation to disambiguate these entries.)

so, our disease ontology, therefore, is filled with some of these omim identifiers.  some of these omim identifiers are mapped to a chromosomal locus, but have not yet been attributed to a gene-type of feature.  

NCBI has created identifiers in their system that represent many of these same "features".  in other cases, it is accurate to represent the omim ids as genes, because that is what they are (and are not synonymous with diseases by their definition).

so we are in a conundrum here.  i think the best thing for us to do, in the case of NCBI, is to type those "unknown" or "other" things as sequence_features (because that's the best knowledge that we have).  those things, then, probably ought to show up on the "variants" tab, because the variants tab are sequence features.  we could consider then either adding a "loci" tab, or changing the gene tab to including other large loci like QTL, genetic marker, and genes.  if they are more-specifically defined elsewhere (like HGNC as a genetic_marker), then we'll get that from them.

**Comment by nlwashington** [comment]
@kshefchek please verify that problematic "genes" are fixed in the current load, and close.

**Comment by nlwashington** [comment]
this is going to be an ongoing problem, as many other sources also catalog some omim diseases as genes.  we will open tickets as those bugs are seen.  i've done by best to deal with this in both the OMIM and NCBIGene parsers.
